supabase/migrations/20240620063026_policy_add_address_permissions.sql

```diff
@@ -0,0 +1,16 @@
+alter table addresses enable row level security;
+
+create policy "User can create addresses"
+on addresses for insert
+to authenticated
+with check ( (select auth.uid()) = user_id );
+
+create policy "User can see their own addresses"
+on addresses for select
+to authenticated
+using ( (select auth.uid()) = user_id );
+
+create policy "User can update their own addresses only"
+on addresses for update
+to authenticated
+using ( (select auth.uid()) = user_id );
```

Comment:
This among other policies have been flagged by the Supabase Performance Advisor page. Not too sure what is causing this as it seems we are using the correct syntax. Think I will seed these tables in later pull requests and come back to this.

![image](https://github.com/codesydney/marketplace-app-for-good/assets/8443215/88296592-782f-44ea-99e8-9d74c30a19aa)


http://localhost:54323/project/default/database/performance-advisor?preset=WARN